ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml

```diff
@@ -55,6 +55,11 @@ tests:
   - mount_path: /etc/sonarcloud/
     name: acm-sonarcloud-token
 - as: fast-forward
+  base_images:
```

Comment:
The repo-brancher entry needs to be part of the base_images stanza that starts on line 1.